docs/operations/batch-processing.md

```diff
@@ -309,3 +310,23 @@ Currently, JanusGraph supports vertices registration for batch processing inside
 step, but not between those local traversals. Also, JanusGraph doesn't register start of the `match` step with any 
 of the local traversals of the `match` step. Thus, performance for `match` step might be limited. This is a temporary 
 limitation until this feature is implemented ([see issue #3788](https://github.com/JanusGraph/janusgraph/issues/3788)).
+
+#### Batch processing for properties
+
+Some of the Gremlin steps with enabled optimization may prefetch vertex properties ib batches. 
+As for now JanusGraph uses slice queries to query part of the row data. A single slice query contains 
```

Comment:
(nit)
```suggestion
As for now, JanusGraph uses slice queries to query part of the row data. A single-slice query contains 
```

Reply:
Thank you. I fixed all these misspells and pushed them